### ISSUE TITLE
Panik at ParseIntError, Kind: InvalidDigit

### ISSUE THREAD

**Issue by Bwahharharrr** [MrGunflame/ts3-rs#3]
When I leave my simple event listening println! bot running, i get alot of:
```rust
thread 'tokio-runtime-worker' panicked at 'called `Result::unwrap()` on an `Err` value: ParseIntError { Kind: InvalidDigit }', /src/event.rs:35:75
```

Not sure if anyone else has seen this or has any ideas?

**Comment by MrGunflame** [comment]
I believe this might have to do with a different `client_servergroups` value separation and should only happen if a client has more than one server group.

**Comment by MrGunflame** [comment]
I believe this is unfortunately not fixable without introduction a breaking change. A fix will be included in `0.4.0`.

**Comment by MrGunflame** [comment]
This should be fixed in `0.4.0`.
